parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -546,6 +548,14 @@ public FileMetaData visit(RangeMetadataFilter filter) throws IOException {
     return parquetMetadata;
   }
 
+  public ParquetMetadata readParquetMetadata(FSDataInputStream from)
+      throws IOException {
+    FileMetaData fileMetaData = CompatibilityUtil.read(from, new FileMetaData());
+    if (Log.DEBUG) LOG.debug(fileMetaData);
+    ParquetMetadata parquetMetadata = fromParquetMetadata(fileMetaData);
+    if (Log.DEBUG) LOG.debug(ParquetMetadata.toPrettyJSON(parquetMetadata));
+    return parquetMetadata;
+  }
```

Comment:
what is this for. This class does not deal with hadoop files so this is probably not the right place to put it.